kdbg/winstack.cpp

```diff
@@ -96,14 +95,14 @@ void WinStack::activateFile(const QString& fileName)
 bool WinStack::activatePath(QString pathName, int lineNo, const DbgAddr& address)
 {
     // check whether the file is already open
-    SourceWindow* fw = 0;
+    SourceWindow* fw = Q_NULLPTR;
```

Comment:
Do not make unrelated changes. At any rate, this should be `nullptr`, not `Q_NULLPTR`.

Reply:
> Do not make unrelated changes. At any rate, this should be `nullptr`, not `Q_NULLPTR`.

I can revert it to 0, but Qt defines Q_NULLPTR for nullptr, so I thought to be consistent.

Reply:
There is no `Q_NULLPTR` in this code base. I prefer not to use Qt's help where possible, and this is such a place.

Best is to leave this line unchanged, as it is not about the topic of your change.

Reply:
Ok, I will revert this too.